model/config.go

```diff
@@ -993,6 +994,10 @@ func (s *ExperimentalSettings) SetDefaults() {
 	if s.EnableRemoteClusterService == nil {
 		s.EnableRemoteClusterService = NewBool(false)
 	}
+
+	if s.EnableAppBar == nil {
+		s.EnableAppBar = NewBool(false)
```

Comment:
Note that this will make it difficult to "enable by default" later. I suppose we will just remove the config variable when we want to permanently enable the feature.

Reply:
Yes, that's the plan.